app/controllers/logs_controller.rb

```diff
@@ -72,6 +72,22 @@ def destroy
 
   end
 
+  def save_draft
+      @log.skip_activation = true
+
+    if @log.update(log_params)
+      respond_to do |format|
+        format.html { redirect_to root_path, notice: "Draft saved for later." }
```

Comment:
I think you could drop the `format.html` responses since we are only updating via js.